examples/docker/shiny/train.py

```diff
@@ -0,0 +1,83 @@
+import os
+import argparse
+from datetime import datetime
+
+import tensorflow as tf
+from sklearn.model_selection import train_test_split
+
+from pymongo.mongo_client import MongoClient
+from pymongo.server_api import ServerApi
+
+URI = os.environ.get("MONGODB_CONNECTION_URI", "")
+
+# Create a new client and connect to the server
+client = MongoClient(URI, server_api=ServerApi('1'))
+
+# Send a ping to confirm a successful connection
+try:
+    client.admin.command('ping')
+    print("Successfully connected to MongoDB!")
+except Exception as e:
+    raise ConnectionError(str(e)) from e
+
+db = client.myDatabase
+my_collection = db["accuracy_scores"]
+
+
+class TrackLossandAccuracyCallback(tf.keras.callbacks.Callback):
+
+    def __init__(self, model_name):
+        self.model_name = model_name
+
+    def on_epoch_end(self, epoch, logs=None):
+        my_collection.insert_one({'model': self.model_name, 'score': logs["accuracy"], 'timestamp': datetime.now()})
```

Comment:
add a print statement here so users can compare what they see in the terminal with what they see in the app

Reply:
[fixed](https://github.com/ploomber/doc/pull/105/files#diff-de90239477773a3e77cf5636a10f5db4b7b1bab1331462d3f5330d480173cf4cR35-R36)